pkgs/development/compilers/dotnet/update.sh

```diff
@@ -297,17 +297,32 @@ Examples:
     major_minor=$(sed 's/^\([0-9]*\.[0-9]*\).*$/\1/' <<< "$sem_version")
     content=$(curl -sL https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/"$major_minor"/releases.json)
     major_minor_patch=$([ "$patch_specified" == true ] && echo "$sem_version" || jq -r '."latest-release"' <<< "$content")
+    major_minor_underscore=${major_minor/./_}
 
     release_content=$(release "$content" "$major_minor_patch")
     aspnetcore_version=$(jq -r '."aspnetcore-runtime".version' <<< "$release_content")
     runtime_version=$(jq -r '.runtime.version' <<< "$release_content")
     sdk_version=$(jq -r '.sdk.version' <<< "$release_content")
 
+    # If patch was not specified, check if the package is already the latest version
+    # If it is, exit early
+    if [ "$patch_specified" == false ] && [ -f "./versions/${sem_version}.nix" ]; then
+        current_version=$(nix-instantiate --eval -E "(import ./versions/${sem_version}.nix { \
+            buildAspNetCore = { ... }: {}; \
+            buildNetSdk = { ... }: {}; \
+            buildNetRuntime = { version, ... }: version; \
+            icu = null; }).runtime_${major_minor_underscore}" | jq -r)
```

Comment:
I think it mmakes more sense to check against the SDK version instead of the runtime, since this script will only work from the SDK packages.

Also, I don't think `jq` does anything here, since we should just be printing the raw version?
```suggestion
            icu = null; }).sdk_${major_minor_underscore}")
```

Reply:
`jq` is needed to remove the quotes from the output of `nix-instantiate`, otherwise the comparision always fails. Unless theres some flag to `nix-instantiate` I don't know about?

Reply:
Ah, that makes sense. I'm personally used to `nix eval` which has a `--raw` flag, but it doesn't look like `nix-instantiate` has any equivalent.